protostar/starknet/cheatable_execute_entry_point.py

```diff
@@ -73,7 +75,9 @@ def _run(
 
         # Run the specified contract entry point with given calldata.
         with wrap_with_stark_exception(code=StarknetErrorCode.SECURITY_ERROR):
-            runner = CairoFunctionRunner(program=contract_class.program, layout="all")
+            runner = CheatableCairoFunctionRunner(  # <-- MODIFICATION # TODO
+                program=contract_class.program, layout="all"
+            )
```

Comment:
TODO?

Reply:
I've seen we have used this convention in other cheatables, we can change it though.

Reply:
This TODO is something is my leftover, i forgot to removed looong time ago. Sorry about that, this TODO should be removed 😅